### ISSUE TITLE
Adding Lars Optimizer to TF addons

### ISSUE THREAD

**Issue by abhinavsp0730** [tensorflow/addons#2337]
**Describe the feature and the current behavior/state.**

**Relevant information**
- Are you willing to contribute it (yes/no):
- Yes
- Are you willing to maintain it going forward? (yes/no):
- Yes
- Is there a relevant academic paper? (if so, where):
- Yes, https://arxiv.org/abs/1708.03888 (It also crosses the 100 citations threshold as discussed in #2194 )
- Is there already an implementation in another framework? (if so, where):
- Was it part of tf.contrib? (if so, where):
- Yes,  https://github.com/tensorflow/tensorflow/blob/r1.15/tensorflow/contrib/opt/python/training/lars_optimizer.py

**Which API type would this fall under (layer, metric, optimizer, etc.)**
Optimizer

CC: @bhack

**Comment by tomerk** [comment]
Notes from ecosystem review: This is not on tf-core's roadmap, addons seems like a reasonable place.

**Comment by bhack** [comment]
Ok you can try to prepare a PR

**Comment by jkr26** [comment]
+1 to this, happy to help on any work that can be factored out

**Comment by abhinavsp0730** [comment]
> +1 to this, happy to help on any work that can be factored out

Thanks @jkr26 I'm open for Colabration for this PR.
But right now I'm facing some difficulty for setting up the dev environment.
It'll take 2-3 days for me to fix my env issue.
Then I'll get back to you for discussing how we're going to colab for this PR.

**Comment by Harsh188** [comment]
I'm also open to collab, let me know if you need any assistance!

**Comment by acarl005** [comment]
We would love to see this as well. LAMB isn't working for us.

**Comment by seanpmorgan** [comment]
TensorFlow Addons is transitioning to a minimal maintenance and release mode. New features will not be added to this repository. For more information, please see our public messaging on this decision:
[TensorFlow Addons Wind Down](https://github.com/tensorflow/addons/issues/2807)

Please consider sending feature requests / contributions to other repositories in the TF community with a similar charters to TFA:
[Keras](https://github.com/keras-team/keras)
[Keras-CV](https://github.com/keras-team/keras-cv)
[Keras-NLP](https://github.com/keras-team/keras-nlp)
